business/istio_config.go

```diff
@@ -1123,6 +1124,24 @@ func (in *IstioConfigService) IsGatewayAPI() bool {
 	return in.k8s.IsGatewayAPI()
 }
 
+func (in *IstioConfigService) IsAmbientProfile() bool {
```

Comment:
The name of this function infers that it will return true if Istio was installed with the ambient *profile* - but that's technically not what we care about, right? I mean, in the future there could be other profiles that also enable Ambient (not just the `ambient` profile). I would call this something like `IsAmbientEnabled` or `IsAmbientSupported`. This really isn't necessarily related to the profile - it is related to the fact that Ambient is enabled in the cluster.

Reply:
I have renamed it for `IsAmbientEnabled`, it is more clear, and you are right, it doesn't need to reflect how it is set, just if the feature is enabled or not. But I think this is one of the things that could change in the future, I mean, maybe this check won't be needed as it could be the default working mode in the mesh. Let's see how it evolves, but I think it should be revisited in future versions.